pkg/codegen/dotnet/gen_program.go

```diff
@@ -374,7 +375,7 @@ type programUsings struct {
 }
 
 func (g *generator) usingStatements(program *pcl.Program) programUsings {
-	systemUsings := codegen.NewStringSet("System.Collections.Generic")
+	systemUsings := codegen.NewStringSet("System.Linq", "System.Collections.Generic")
```

Comment:
Do we need to add Linq all the time? Can we trim this to just when we need Linq methods easily?

Reply:
> Do we need to add Linq all the time?

No

>  Can we trim this to just when we need Linq methods easily?
not easily, would need to detect functions and range expressions that emit a `.Select(...)` in them